common/openapi.yml

```diff
@@ -258,8 +258,52 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/policy/v1alpha1/region/{objectId}/evaluate:
+    parameters:
+      - $ref: '#/components/parameters/TpsObjectId'
+      - $ref: '#/components/parameters/Datacenter'
+      - $ref: '#/components/parameters/Platform'
+    get:
+      summary: Evaluate whether a datacenter is available inside of a region constraint
+      operationId: evaluateRegionDatacenter
+      tags: [Tps]
+      responses:
+        '200':
+          description: Constraint evaluated successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TpsPaoGetResult'
+        '204':
+          description: Datacenter evaluated successfully
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '403':
+          $ref: '#/components/responses/PermissionDenied'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
 components:
   parameters:
+    Datacenter:
+      name: datacenter
+      in: query
+      description: The TPS id of a datacenter
+      required: true
+      schema:
+        type: string
+
+    Platform:
+      name: platform
+      in: query
+      description: The cloud platform (gcp, azure)
+      required: true
+      schema:
+        type: string
+        enum: ["gcp", "azure", "aws"]
```

Comment:
nit: I don't think we have any aws regions in the config atm, do we need to include that as an enum value here?